lnrpc/rpc.proto

```diff
@@ -839,6 +849,22 @@ message LightningAddress {
     string host = 2 [json_name = "host"];
 }
 
+message EstimateFeeRequest {
+    /// The map from addresses to amounts for the transaction.
+    map<string, int64> AddrToAmount = 1;
+
+    /// The target number of blocks that this transaction should be confirmed by.
+    int32 target_conf = 2;
+}
+
+message EstimateFeeResponse {
+    /// The total fee in satoshis.
+    int64 fee_sat = 1;
+
+    /// The fee rate in satoshi/byte.
+    int64 feerate_sat_per_byte = 2;
```

Comment:
Nit: rename to `fee_rate`?

Reply:
would say perhaps _just_ `sat_per_byte`. we already know it's a feerate because of the rpc name, i think the units are more descriptive/important to the uesr

Reply:
porque no los dos?

Reply:
The name is just slightly too long IMO.